.circleci/config.yml

```diff
@@ -72,5 +72,5 @@ jobs:
               bin/ci/deploy-dockerhub.sh "circle_build_$CIRCLE_BUILD_NUM"
             fi
             if [ -n "${CIRCLE_TAG}"]; then
```

Comment:
Shame you can't run shellcheck on this circle config file. Maybe this is complex enough it's worth moving to a shell script?